### DISCUSSION TITLE
v1/search endpoint search bug via unity package manager

### DISCUSSION THREAD

**Discussion by ozanbatuhanozdal** [verdaccio/verdaccio#3480] [Q&A] [answered]
Hello, i am using verdaccio as a private package registry have a access via google authentication we can use web search with 0 errors but when using unity via package manager we cant find our packages in unity and get this error 
![Screenshot 2022-11-07 at 11 01 34](https://user-images.githubusercontent.com/31401985/200260987-6405084e-3e9e-4a01-8388-73af6580a104.png)

**Comment by juanpicado** [comment, marked as answer]
Not sure what might be but would be great have more context.
